helm/captcha/values.yaml

```diff
@@ -279,8 +288,8 @@ volumePermissions:
   enabled: false
   image:
     registry: docker.io
-    repository: bitnami/bitnami-shell
-    tag: "10"
+    repository: mosipid/os-shell
+    tag: "12-debian-12-r46"
     pullPolicy: Always
     ## Optionally specify an array of imagePullSecrets.
     ## Secrets must be manually created in the namespace.
```

Comment:
repository: mosipid/captcha-validation-service
  tag: 0.1.0-beta.1 change to   repository: mosipqa/captcha-validation-service
  tag: 0.1.x